tsconfig.json

```diff
@@ -57,6 +57,11 @@
     // "experimentalDecorators": true,        /* Enables experimental support for ES7 decorators. */
     // "emitDecoratorMetadata": true,         /* Enables experimental support for emitting type metadata for decorators. */
   },
+  /** VSCode won't recognize these types */
+  "files" : [
+    "src/schedule.d.ts",
+    "src/workflow.d.ts"
+  ],
```

Comment:
@yokotaso VS Code and users of this package needs `/// <reference />`, this config file is only for internal developers of this package.